RadixWallet/SargonExtensions/SargonHostInteractor.swift

```diff
@@ -0,0 +1,107 @@
+import Sargon
+
+final class SargonHostInteractor: HostInteractor {
+	@Dependency(\.overlayWindowClient) var overlayWindowClient
+	@Dependency(\.continuousClock) var clock
+
+	func signTransactions(request: SargonUniFFI.SignRequestOfTransactionIntent) async throws -> SargonUniFFI.SignResponseOfTransactionIntentHash {
```

Comment:
maybe `typealias Request = SargonUniFFI.SignRequestOfTransactionIntent` ? and `typealias Response = SargonUniFFI.SignResponseOfTransactionIntentHash` ?